README.md

```diff
@@ -73,8 +83,8 @@ For more information about inheriting configuration from a gem please check
 * Don't use `;` to separate statements and expressions. As a corollary - use one
   expression per line.
 
-* Use spaces around operators, after commas, colons and semicolons, around
```

Comment:
Are these formatting changes accidental?

Reply:
Sorry for including this. I fixed oversized lines in the same branch before changing the content. I put this in a separate commit for easier review 🙏🏼